### ISSUE TITLE
Have you tried training EfficientDet with EfficientNet weights？

### ISSUE THREAD

**Issue by lxyyang** [zylo117/Yet-Another-EfficientDet-Pytorch#46]
I try training EfficientDet with EfficientNet weights， but after 3 epoch ，the ap of coco is zero！

**Comment by zylo117** [comment]
Please share loss graph, including training loss, val loss, logs.

**Comment by lxyyang** [comment]
@zylo117 
Val. Epoch: 4/500. Classification loss: 1.03457. Regression loss: 0.14781. Total loss: 1.18238
Step: 12499. Epoch: 5/500. Iteration: 180/2464. Cls loss: 1.03119. Reg loss: 0.15889. Total loss: 1.19008

**Comment by zylo117** [comment]
You can monitor loss graph using tensorboard. I think it is still at the very early stage. It takes weeks to train a effdet d0. If you're training D3/4 or bigger network, it takes longer.

**Comment by lxyyang** [comment]
@zylo117 
you write at README，
 “# This takes time and requires change 
# of hyperparameters every few hours.”
so how change the hyperparameters ？
i used “python train.py -c 0 --batchsize 48”。

**Comment by zylo117** [comment]
check all arguments by
`python train.py -h`

**Comment by xingyueye** [comment]
> @zylo117
> Val. Epoch: 4/500. Classification loss: 1.03457. Regression loss: 0.14781. Total loss: 1.18238
> Step: 12499. Epoch: 5/500. Iteration: 180/2464. Cls loss: 1.03119. Reg loss: 0.15889. Total loss: 1.19008

@lxyyang hi， have you test ap while total loss is 1.1...
In my test, I could only get a poor ap 0.012 while on Coco, while train loss is blow 0.6?
Is there some thing wrong in eval phase?

**Comment by zylo117** [comment]
val cls loss for D2 is 0.25, val reg loss is 0.025. So I think you still have a long way to go.
This is because coco is too hard for training, but if you finetune it in a custom dataset, it should be within a day or two to get a decent result.
